modules/docs/arrow-docs/docs/js/docVersionsToogle.js

```diff
@@ -0,0 +1,49 @@
+/* When the user clicks on the navigation Documentation button,
+toggle between hiding and showing the dropdown content */
+function displayToogle() {
+  document.getElementById("docVersionDropdown").classList.toggle("show");
+}
+
+// Close the dropdown if the user clicks outside of it
+window.onclick = function(event) {
```

Comment:
This is a bit hacky. Overriding global objects could lead to other parts of the website going bananas, never knowing why. Also we are penalizing each and every click on the page for a simple dropdown closing.

It would be better if we implement what is trying to be achieved here through the lose of focus on the specific element that requires it, or if adding some code to a the global `onclick` event is needed, the best way would be through [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener).